### PR TITLE
feat: #1104 Align design for User Card component

### PR DESCRIPTION
**Pull request by algrinkevich**

### CODE REVIEW

File: mw-webapp/src/component/userCard/UserCard.tsx [hunk 1]
```diff
@@ -109,15 +109,25 @@ export const UserCard = observer((props: UserCardProps) => {
...
+            <span>
+              {`${LanguageService.allUsers.userCard.createdAt[language]}`}
+            </span>
+            <span className={styles.createdAt}>
+              {DateUtils.getShortISODotSplitted(props.userPreview.createdAt)}
+            </span>
+          </HorizontalContainer>
+          <HorizontalContainer className={styles.ways}>
+            <span>
+              {`${LanguageService.allUsers.userCard.ownWays[language]} (${props.userPreview.ownWays})`}
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/userCard/UserCard.tsx:122]: Is it possible don't use span wrappers for each element?

File: mw-webapp/src/globalStore/ThemeStore.ts [hunk 2]
```diff
@@ -53,6 +53,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#8F8F8F",
     [Theme.LIGHT]: "#A8A1DD",
   },
+  secondaryCardsBgColor: {
```
Comment (Ekaterina1994) [inline mw-webapp/src/globalStore/ThemeStore.ts:56]: do we have primaryCradsBgColor? If not then name is not actual. You can use name for example additionalInfoCardsBgColor

Comment (algrinkevich) [reply mw-webapp/src/globalStore/ThemeStore.ts]: No, we don't have the primary. I renamed to  `additionalInfoCardBgColor`

File: mw-webapp/src/component/userCard/UserCard.tsx (hunk 1 above)
Comment (algrinkevich) [reply mw-webapp/src/component/userCard/UserCard.tsx:122]: If I remove the wrappers, there will be a small distance between "ways", not the same as in the design.

Design:
image

Without wrappers:
image

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**